tests/integration_tests/test_autopilot_prediction_udf_real.py

```diff
@@ -42,6 +42,7 @@ def get_connection(self, name: str) -> Connection:
 class Context:
     def __init__(self):
         self._emitted = []
+        self._is_emitted_once = False
```

Comment:
I am not sure, but maybe the name is a bit confusing here.
Maybe better:
```suggestion
        self._is_accessed_once = False
```